frontend/components/base/buttons/filter-button.tsx

```diff
@@ -92,24 +90,24 @@ export default function FilterButton({
   return (
     <Popover open={open} onOpenChange={handleOpenChange}>
       <PopoverTrigger asChild>
-        <Button variant={variant} className={cn('h-8 py-2 px-3 border-dashed text-sm text-foreground', className)}>
+        <Button variant={variant} size="sm" className={cn('border-dashed', className)}>
           {icon ? icon : <PlusCircle className="" />}
           <span>{label}</span>
-          {!actionMode && selectedCount > 0 && <Badge className="w-[20px] h-[20px] items-center justify-center">{selectedCount}</Badge>}
+          {!actionMode && selectedCount > 0 && (
+            <Badge className="size-5 items-center justify-center">{selectedCount}</Badge>
+          )}
         </Button>
       </PopoverTrigger>
       <PopoverContent className="w-[229px] max-h-[240px] p-0 flex flex-col" align="start">
         <Command className="flex flex-col">
           <CommandInput
             placeholder={placeholder || label || 'Search...'}
             leftAddon={<Search size={16} className="text-muted-foreground mr-1" />}
-            className="rounded-none"
-            wrapperClassName="shrink-0 h-[40px] border-t-0 border-l-0 border-b-1 border-r-0 border-color-border rounded-none"
```

Comment:
Should have been fixed in the Command component since the command component styling was wrong. PR done: https://github.com/radiant-network/radiant-portal/pull/381